lardataobj/RecoBase/classes_def.xml

```diff
@@ -112,10 +112,12 @@
     <version ClassVersion="11" checksum="4046040025"/>
   </class>
   <class name="recob::OpWaveform" ClassVersion="12">
-   <version ClassVersion="12" checksum="290846489"/>
+    <version ClassVersion="12" checksum="290846489"/>
     <version ClassVersion="11" checksum="352214401"/>
   </class>
-  <class name="recob::OpHit" ClassVersion="14">
+  <class name="recob::OpHit" ClassVersion="16">
+    <version ClassVersion="16" checksum="1607873362"/>
+    <version ClassVersion="15" checksum="3210767731"/>
```

Comment:
It looks like you tried building `OpHit` multiple times, with different data representations--this yields two new checksums instead of one.  I suggest that you delete these lines, and then rebuild.  You should then get something that looks like:

```suggestion
    <version ClassVersion="15" checksum="1607873362"/>
```